src/library_glew.js

```diff
@@ -100,7 +100,7 @@ var LibraryGLEW = {
 
     extensionIsSupported(name) {
       if (!GLEW.extensions) {
-        GLEW.extensions = UTF8ToString(_glGetString(0x1F03)).split(' ');
+        GLEW.extensions = GLctx.getSupportedExtensions() || [];
```

Comment:
This does not apply the processing that `GL_EXTENSIONS_IN_PREFIXED_FORMAT` does in `library_webgl.js`, so it changes behavior potentially?

https://github.com/emscripten-core/emscripten/blob/f140f5cc1f162bdd3360e77676fa452dde15ea62/src/library_webgl.js#L1143-L1149

Reply:
Good catch.  Fixed by adding new `GL.getExtensions()` helper method.